### ISSUE TITLE
No Connection to Server on self build Portable Instant Support Client 

### ISSUE THREAD

**Issue by Basti-Fantasti** [lucent-sea/Remotely#281]
**Describe the bug**
When starting the self build Portable Windows Support Client (tested 32 and 64 Bit) there's no connection possible to the preconfigured server. Server configuration is correct. I tested it using the same URL, same Server instance and original Remotely_Desktop which came with the Ubuntu Install script without custom build.

Update:
It also works using the custom build Windows Installer (I don't know if this is really custom build because Server instance ID and server address need to be specified). But right after installation the service is started and the PC visible in the admin backend.

## Self built version

![2021-04-15 11_24_40-Window](https://user-images.githubusercontent.com/1663072/114846688-36f08e00-9ddd-11eb-96c6-b76e854e1b2a.png)

![2021-04-15 11_25_19-Window](https://user-images.githubusercontent.com/1663072/114846790-5091d580-9ddd-11eb-9364-714816187b33.png)

## Original not customized version

![2021-04-15 11_27_09-Window](https://user-images.githubusercontent.com/1663072/114847106-a23a6000-9ddd-11eb-978e-cb2ccdaf4e12.png)

**To Reproduce**
1. Build the Server for Ubuntu/NGINX
2. Setup Remotely
3. Download customized portable Windows client
4. Start the client
5. See error

**Remotely Version**
Server: 2021.04.14.1320
Agent: 2021.4.14.1320

**Expected Behavior**
Acquire ID from Server

**Desktop:**
 - OS: Windows 10
 - Browser Firefox
 - Version 87.0

**Comment by lucent-sea** [comment]
Should be fixed now.  I'll wait for confirmation.  :)

**Comment by lucent-sea** [comment]
Oh.  I should probably mention that I changed the GitHub build to checkout from my repo, so it doesn't require pulling the upstream into your fork to build the latest changes.

I haven't tested that it works on a different account yet, though.   So hopefully that works without issue.

I also added an option back in to use the pre-built ZIP packages on the releases page, which is how the previous install scripts worked.  It's quicker and easier, but the server URL won't be hard-coded into the clients.
